.github/actions/deploy/action.yml

```diff
@@ -1,6 +1,11 @@
 name: "Deploy"
 description: "Deploy to S3"
 
+inputs:
+  git_sha:
+    description: "git SHA of code being deployed"
+    required: true
+
 runs:
   using: "node12"
```

Comment:
Should we change this to node14 too to match the new Earthly environment? Looks like there's one other reference in another action too.

Reply:
I might be more inclined to change the earthly environment to be node 12, to reduce the number of things I'm changing in this patch